CogniteSdk.Types/Transformations/Configs/TransformationDestination.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright 2022 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Text.Json.Serialization;
+
+namespace CogniteSdk
+{
+    /// <summary>
+    /// Where result of a transformation is written to.
+    /// </summary>
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum TransformationDestinationType
```

Comment:
We should use explicit associated constant values so that no-one e.g makes a PR later that adds a new entry that will change the values of all the following members. I.e `assets = 0` etc

Reply:
Would that matter? We are using the string representation.